services/appmotion_gateway/docker-compose.yml

```diff
@@ -73,6 +73,13 @@ services:
             - WEB_DOCUMENT_ROOT=/app/public
         deploy:
             replicas: 1
+            labels:
+                - traefik.enable=true
+                - traefik.docker.network=${PUBLIC_NETWORK}
+                - traefik.http.routers.appmotion_gateway.rule=Host(`payments.${MACHINE_FQDN}`)
+                - traefik.http.routers.appmotion_gateway.entrypoints=https
+                - traefik.http.routers.appmotion_gateway.tls=true
+                - traefik.http.services.appmotion_gateway.loadbalancer.server.port=80
```

Comment:
Can you add also some protection against last number of requests per second.

Reply:
Added